### ISSUE TITLE
Warning about link-initialize.js script embedded more than once 

### ISSUE THREAD

**Issue by rblakejohnson** [plaid/react-plaid-link#103]
When I go to the page that uses `react-plaid-link` a second time, e.g. I click back and forward in the browser, there's a warning in the console:

```react_devtools_backend.js:6 Warning: The Plaid link-initialize.js script was embedded more than once. This is an unsupported configuration and may lead to unpredictable behavior. Please ensure Plaid Link is embedded only once per page.```

I'd expect the `usePlaidLink` hook to manage this, but I'm probably missing something. Under the hood it looks like `react-plaid-link` is using `useScript` which I believe can manage loading 3rd party scripts efficiently.

**Comment by timruffles** [comment]
Thanks for reporting. Which version of react-plaid-link was this? Are you seeing https://github.com/plaid/react-plaid-link/issues/86, which was fixed in 2.2.1?

**Comment by rblakejohnson** [comment]
@timruffles thanks for your time. I'm not seeing #86, but I'm on react-plaid-link version 2.1.2. Seems odd since I installed it 9 days ago. Anyway, I'll try updating to the latest next week and see if that works.

**Comment by timruffles** [comment]
@rblakejohnson it does sound like the issue identified in #86 and fixed in 2.2.1 - if you update to 2.2.1 and the bug is present please reopen and let us know.

**Comment by rblakejohnson** [comment]
@timruffles I updated to 2.2.1 and I'm still seeing the same issue. Here's a screenshot:
![image](https://user-images.githubusercontent.com/1006869/80653800-2f06d500-8a38-11ea-9146-105220f48316.png)
Also I looked at #86 again and this is a different issue. I've never experience the initial load failing. What I'm seeing is that the Plaid link-initialize.js is loaded more than once if you come back to the page where react-plaid-link is used as seen in the Warning from React
